hw03_grayr2.py

```diff
@@ -0,0 +1,164 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Relly Gray
+# Username: grayr2
+#
+# Assignment:
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1UVBNWf08HXY_rivdGX1H_HSzcBGa33XGG7AeElADZLg/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: ChatGpt
+# I asked chatgpt how it may be possible to turn my shape into a skyscraper and it gave me back this: rell.setpos(-100, 100 - (floor + 1) * floor_height)  # Adjust position for the next floor
+#
+#################################################################################
+
+
+import turtle
+
+rell = turtle.Turtle()
+rell.speed(10)
+
+def skyscraper():
+    """
+    Example docstring for function_1. function_1 is not a good function name and should be changed.
+    """
+    pass
+    rell.color("grey")
+    rell.penup()
+    rell.setpos(-100, 200)
+    rell.pendown()
+    rell.pensize(5)
+    floor = 10
+    floor_height = 50
+    rell.begin_fill()
+    for floor in range(4):
+        rell.fd(200)
+        rell.right(90)
+    rell.penup()
+    rell.setpos(-100, 200 - (floor + 1) * floor_height)  # Adjust position for the next floor
+    rell.pendown()
+    for floor in range (4):
+        rell.fd(300)
+        rell.right(90)
+    rell.end_fill()
+
+def windows():
+    """
+    Example docstring for function_2. function_2 is not a good function name and should be changed.
+    """
+    pass
+    rell.pensize(1)
+    rell.color("lightblue")
+    rell.penup()
+    rell.setpos(-90, 190)
+    rell.pendown()
+    rell.begin_fill()
+    for windows in range(4):
+        rell.fd(40)
+        rell.right(90)
```

Comment:
These seven lines of code appear to repeat multiple times below still. Try and get them into a loop, and figure out how to pass in the correct values to `setpos` each iteration of the loop.